src/osdc/Objecter.cc

```diff
@@ -2909,6 +2908,7 @@ int Objecter::_calc_target(op_target_t *t, Connection *con, bool any_change)
       t->osd = acting_primary;
     }
   }
+  t->acting = std::move(acting);
```

Comment:
This doesn't appear to be the equivalent transformation because the setting of `t->acting` should be conditional on `legacy_change || split_or_merge || force_resend`.  But see my previous comment on `t->acting`.